dart/biomechanics/SubjectOnDisk.cpp

```diff
@@ -720,6 +765,20 @@ s_t SubjectOnDisk::getLowpassCutoffFrequency(int trial, int processingPass)
 // that (Butterworth) filter?
 int SubjectOnDisk::getLowpassFilterOrder(int trial, int processingPass)
 {
+  if (trial >= getNumTrials())
```

Comment:
Check that `trial` is non-zero too? Same with the three other if-statements below.